src/omero_zarr/cli.py

```diff
@@ -163,20 +164,21 @@ def masks(self, args):
 
     @gateway_required
     def export(self, args):
-
         if isinstance(args.object, ImageI):
             image_id = args.object.id
             image = self._lookup(self.gateway, "Image", image_id)
-            self.ctx.out("Export image: %s" % image.name)
+            inplace = image.getInplaceImport()
 
             if args.bf:
-                path, name = self._get_path(image_id)
-                if path:
-                    self._do_export(path, name, args)
-                else:
-                    print(
-                        "Couldn't find managed repository path for this image."
-                    )
+                prx, desc = self.client.getManagedRepository(description=True)
+                repo_path = Path(desc._path._val) / Path(desc._name._val)
+                paths = image.getImportedImageFilePaths()['client_paths']\
+                    if inplace else \
```

Comment:
Do you mind unrolling this into a normal if/else perhaps with a print statement saying which path was chosen?

Reply:
Sure, can do. Just wanted to be proper pythonic and do it in a short and cryptic way.